dfastmi/batch/core.py

```diff
@@ -483,21 +483,22 @@ def _get_root_dir(rootdir: Path) -> Path:
     return rootdir
 
 
-def count_discharges(discharges: Vector) -> int:
+def count_discharges(apply_q: Tuple[bool, bool, bool]) -> int:
     """
-    Count the number of non-empty discharges.
+    Count the number of discharges to be used (apply_q = True).
 
     Arguments
     ---------
-    discharges : Vector
-        Tuple of (at most) three characteristic discharges (Q).
+    apply_q : Tuple[bool, bool, bool]
```

Comment:
update the commented reflecting the new type

Reply:
done